cluster/gce/util.sh

```diff
@@ -22,7 +22,7 @@
 readonly GCE_MAX_LOCAL_SSD=8
 
 KUBE_ROOT=$(dirname "${BASH_SOURCE}")/../..
-source "${KUBE_ROOT}/cluster/gce/${KUBE_CONFIG_FILE-"config-default.sh"}"
+source "${KUBE_ROOT}/cluster/gce/${KUBE_CONFIG_FILE:-"config-default.sh"}"
```

Comment:
This is valid grammar, please don't touch it if there is no necessary reason. you can refer to: https://tldp.org/LDP/abs/html/parameter-substitution.html

Reply:
change back.